scripts/update-apk.sh

```diff
@@ -27,9 +27,11 @@ if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	${ANDROID_HOME}/build-tools/27.0.3/zipalign -v -p 4 app-release-unaligned.apk app-release.apk
 fi
 
-for file in app*; do
-  mv $file test-${file%%}
-done
+if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+    for file in app*; do
+          cp $file eventyay-organizer-master-${file%%}
```

Comment:
This won't do the trick. Check open-event-android and susi_android

Reply:
The fastlane app name below isn't changed

Reply:
@iamareebjamal Changed the name.

Reply:
should it be eventyay-organizer-master?